docker-entrypoint.sh

```diff
@@ -1,61 +1,366 @@
+    success "Update the file ${target_file_path} with ${original_file_path}"
+    cp -f "${original_file_path}" "${target_file_path}" || warning "Could not copy ${original_file_path} to ${target_file_path}"
+    if [ "${print_migration_info_on_update}" = "true" ]; then
+      print_migration_info="true"
+    fi
+  else
+    success "The file ${target_file_path} is up to date"
+  fi
+}
+
+extract_config_info() {
+  local config_location="$1"
+  local config_variable="$2"
+  local config_value=""
+  if [[ "${config_location}" = *.yml ]]; then
+    config_value=$(yq -r "${config_variable}" "${config_location}")
+  elif [[ "${config_location}" = *.json ]]; then
+    config_value=$(jq -r "${config_variable}" "${config_location}")
+  fi
+  # Validate the config value
+  if [ -z "${config_value}" ] || [ "${config_value}" = null ]; then
+    config_value=""
+  fi
+  # Return the value
+  echo "${config_value}"
+}
+
+echo "#################"
+echo "# Container ENV #"
+echo "#################"
+# Validate the log level
+set_log_level
+
+# Set the jar file location
+jar_file=/ors.jar
+BUILD_GRAPHS=${BUILD_GRAPHS:-"false"}
+REBUILD_GRAPHS=${REBUILD_GRAPHS:-"false"}
+# If BUILD_GRAPHS is set to true, we need to set ors_rebuild_graphs to true and print an info about migration to REBUILD_GRAPHS
+if [ "${BUILD_GRAPHS}" = "true" ]; then
+  ors_rebuild_graphs="true"
+  warning "BUILD_GRAPHS is deprecated and will be removed in the future."
+  warning "Please use REBUILD_GRAPHS instead."
+elif [ "${REBUILD_GRAPHS}" = "true" ]; then
+  ors_rebuild_graphs="true"
+else
+  ors_rebuild_graphs="false"
+fi
+
+
+# get ors.engine.graphs_root_path=. Dot notations in bash are not allowed, so we need to use awk to parse it.
+ors_engine_graphs_root_path=$(env | grep "^ors\.engine\.graphs_root_path=" | awk -F '=' '{print $2}')
+# get ors.engine.elevation.cache_path
+ors_engine_elevation_cache_path=$(env | grep "^ors\.engine\.elevation\.cache_path=" | awk -F '=' '{print $2}')
+# get ors.engine.source_file
+ors_engine_source_file=$(env | grep "^ors\.engine\.source_file=" | awk -F '=' '{print $2}')
+# get ors_config_location
+ors_config_location=${ORS_CONFIG_LOCATION:-""}
+# Unset the ORS_CONFIG_LOCATION to not interfere with the spring-boot application
+unset ORS_CONFIG_LOCATION
+# Parse the ors.* properties
+env | while read -r line; do
+  debug "${line}"
+done
+info "Any config file settings can be overwritten by environment variables."
+info "Use 'CONTAINER_LOG_LEVEL=DEBUG' to see the full list of active environment variables for this container."
+
+echo "###########################"
+echo "# Container sanity checks #"
+echo "###########################"
+info "Running container as user $(whoami) with id $(id -u) and group $(id -g)"
+if [[ $(id -u) -eq 0 ]] || [[ $(id -g) -eq 0 ]] ; then
+  debug "User and group are set to root with id 0 and group 0."
+elif [[ $(id -u) -eq 1000 ]] || [[ $(id -g) -eq 1000 ]] ; then
+  debug "User and group are set to 1000 and group 1000."
+else
```

Comment:
Given the line above, this seems redundant?

Reply:
The goal here is more to react with an explicit debug message on both scenarios individually. Either 0 or 1000 have different implications for the runtime container.

Reply:
Can these implications be easily mentioned? If so, we might want to do that.
If not, that's fine by me.

Reply:
Nope. It depends on how they read the documentation and what way they set it up. When people temper with permissions, the container needs to feed back what it is ultimately using. Else it'll become a black box in that regard.